signing/sigstore/rekor.go

```diff
@@ -88,6 +88,10 @@ func (r *Rekor) Upload(ctx context.Context, att signing.Attestation) (signing.Lo
 	}
 
 	fmt.Printf("Uploaded signed attestation to rekor with UUID %s.\n", uuid)
+	fmt.Printf(`You could use rekor-cli to view the log entry details:"
+	"$ rekor-cli get --uuid %s --rekor_server %s",
+	"In addition to that, you could also use the Rekor Search UI:"
+	"https://search.sigstore.dev/?uuid%s`, uuid, r.rekorAddr, uuid)
```

Comment:
I think the '"' double quotes will get printed in a raw string like this.
See: https://go.dev/play/p/6hG4rHMvJ6w

I think you need to remove the double quotes and the whitespace at the beginning of the line.

Also, for now I think it doesn't matter, but the second half is only relevant if using the public rekor instance. Maybe something like the following? (https://go.dev/play/p/7ZbUsexpgIa)

```suggestion
	fmt.Printf("You could use rekor-cli to view the log entry details:\n\n"+
		"  $ rekor-cli get --uuid \"%s\" --rekor_server \"%s\"\n\n", uuid, r.rekorAddr)
	if r.rekorAddr == DefaultRekorAddr {
		fmt.Printf("In addition to that, you could also use the Rekor Search UI: \n\n"+
			"  https://search.sigstore.dev/?uuid=%s\n\n", uuid)
	}
```

Reply:
or for the sake of simplicity, we may ignore adding the Rekor server address to the log message.

Reply:
I've updated accordingly: https://go.dev/play/p/EnhhKXMjRKv